examples/nextjs-with-typescript/pages/MuxUploader.tsx

```diff
@@ -39,10 +39,7 @@ function MuxUploaderPage() {
           style={{ display: "block", height: "200px" }}
           id="uploader"
           endpoint={url}
-          // Uncomment for example of custom progress formatting.
-          // formatProgress={(percent: number) => `${percent} percent uploaded`}
           type="bar"
```

Comment:
nitpick(blocking): type is no longer a functional attribute on <mux-uploader>. Remove (and all other cases).